src/components/Actions/Actions.vue

```diff
@@ -57,7 +57,7 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 	<!-- if only one action, check if we need to bind to click or not -->
 	<element v-if="isValidSingleAction && !forceMenu"
 		v-tooltip.auto="firstAction.text"
-		:aria-label="firstAction.text"
+		:aria-label=" !!ariaLabel ? ariaLabel : firstAction.text"
```

Comment:
```suggestion
		:aria-label="ariaLabel || firstAction.text"
```

Reply:
Are you sure??? Wouldn't that return true or false??

Reply:
Open your browsers console and do:
```
var a = ''
var b = 'bar'
a || b

> "bar"
```